GameData/SPEngine/Parts/A4_proc.cfg

```diff
@@ -12,7 +12,21 @@
 	!engineType = // prevent ROE from deleting us
 }
 
-+PART[Bumper_Engine_Unclad]:AFTER[ZREALPLUME]
++PART[Bumper_Engine_Unclad]:NEEDS[!ROEngines]:AFTER[ZREALPLUME]
+{
+	@name = SPEngineA4Unclad
+	@title = A-class Procedural Engine (Unclad)
+	@description = Alcohol/LOx atmospheric engine, like Aggregat or RD-100 series.  Without enclosing boattail.
+	MODULE
+	{
+		name = ModuleSPEngine
+		familyLetter = A
+	}
+	@identicalParts = Bumper_Engine,Bumper_Engine_Unclad
+	!engineType = // prevent ROE from deleting us
+}
+
++PART[ROE-A4]:NEEDS[ROEngines]:AFTER[ZREALPLUME]
 {
 	@name = SPEngineA4Unclad
 	@title = A-class Procedural Engine (Unclad)
```

Comment:
This part's `@description` should really have a `<b><color=green>From ROEngines mod</color></b>`.
(Unfortunately I can't put the comment on the correct line, because GitHub only lets me comment on lines of patch/context, rather than the whole file.)